mmv1/products/cloudrun/terraform.yaml

```diff
@@ -86,6 +86,14 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           cloud_run_service_name: "cloudrun-srv"
         test_env_vars:
           project: :PROJECT_NAME
+      - !ruby/object:Provider::Terraform::Examples
+        name: "cloud_run_service_vpc_connector"
+        primary_resource_id: "default"
+        primary_resource_name: "fmt.Sprintf(\"tf-test-cloudrun-srv%s\", context[\"random$
```

Comment:
Looks like this line got cut off and no longer compiles

Reply:
fixed